<launch>
    <rosparam file="$(find butia_world)/config/redis.yaml" ns="redis" command="load" />
    <node pkg="butia_world" type="object_recognition_writer.py" name="object_recognition_writer" output="screen"> 
    </node>
</launch>